Dockerfile

```diff
@@ -57,12 +61,8 @@ RUN bundle exec bootsnap precompile app/ lib/
 # container.
 
 RUN /bin/bash -c 'if [[ "$RAILS_ENV" == "production" ]]; then \
-      mv config/credentials.yml.enc config/credentials.yml.enc.backup && \
       mv config/credentials/production.yml.enc config/credentials/production.yml.enc.backup && \
-      mv config/credentials/sample.yml.enc config/credentials.yml.enc && \
-      mv config/credentials/sample.key config/master.key && \
-      SECRET_KEY_BASE_DUMMY=1 ./bin/rails assets:precompile && \
-      mv config/credentials.yml.enc.backup config/credentials.yml.enc && \
+      SECRET_KEY_BASE_DUMMY=1 RAILS_ENV=development ./bin/rails assets:precompile && \
```

Comment:
We should add a `TODO` comment to revisit this workaround and also make sure we don't leave the hard-coded `development` env in there.